Cargo.toml

```diff
@@ -24,4 +24,7 @@ lto = true
 opt-level = "s"
 
 [dev-dependencies]
+error-chain = "0.12.2"
+futures = "0.3"
 mockall = "0.8.0"
```

Comment:
These libs are used in mackerel_client. In production code, they ain't used. So, I add these in dev-dependencies.